src/lib/client.js

```diff
@@ -50,6 +53,7 @@ StreamClient.prototype = {
     this.group = this.options.group || 'unspecified';
     // track subscriptions made on feeds created by this client
     this.subscriptions = {};
+    this.noJWTTimestamp = this.options.noJWTTimestamp || true;
```

Comment:
I would rename noJWTTimestamp (keep JWT out of it) to something like expireTokens or something similar.